addons/hr_expense/static/tests/mobile/expense_mobile.test.js

```diff
@@ -0,0 +1,25 @@
+import { describe, expect, test } from "@odoo/hoot";
+import { openListView, registerArchs, start } from "@mail/../tests/mail_test_helpers";
+import { defineHrExpenseModels } from "../mock_server/mock_models/defineModel";
+import { tick } from "@odoo/hoot-mock";
+
+describe.current.tags("mobile");
+defineHrExpenseModels();
+
+test("expense dashboard can horizontally scroll", async () => {
+    const newArchs = {
```

Comment:
useless variable as it is only used by `registerArchs`

Reply:
I moved it out of the test scope so it can be reused by other test if needed.